### DISCUSSION TITLE
RE7 and RE8 features

### DISCUSSION THREAD

**Discussion by XavierHG** [praydog/REFramework#315] [General Questions]
First, thank you very much to the developers. Congratulations!
This is just wonderful, I love horror games and always wanted RE7 and RE8 in VR (also Outlast games)
Although this mod is great there are 3 things that do not allow me to enjoy the experience.

**- The lack of headset VR move direction.**

**- The lack of Snap turning** gives me motion sickness. If I try to turn my body instead, it happens that my body is looking in one 
direction but the in-game body remains in another, therefore I walk in a different direction than I am facing.

**- In-game body.** If I step my body forward or backward, the in-game body gets separates from me, I see it in front of behind me, it does not stay with me.

I know the mod is under development yet, will all this be fixed? 
Is currently any options that can help me to improve some of this?

**Comment by praydog** [comment]
It's not released yet. Wait until it's released, which will be soon.

**Reply by XavierHG** [reply]
Ok, I guess the features I mentioned will be present at release date...
Thanks.
